src/asset-files.ts

```diff
@@ -31,7 +32,7 @@ export async function serveAsset(
   if (!file) return
 
   return {
-    data: await file.content(),
+    data: Readable.from(await file.content()),
```

Comment:
Not sure what to do here, AssetFile does not feature the typical `Blob`-style stream API, so we still need to read the contents first and then create a `Readable` from that.

Reply:
Uses `fs.createReadStream` now